README.md

```diff
@@ -9,35 +9,38 @@ Welcome! This repo is intended to highlight some awesome Dart Frog resources —
 
 [Dart Frog](https://github.com/VeryGoodOpenSource/dart_frog) is a fast, minimalistic backend framework for Dart built by [Very Good Ventures](https://verygood.ventures/). It can increase efficiency by allowing you to share code, tooling, and processes across the front and backend.
 
-
```

Comment:
Can you revert the changes (which were probably automatically applied by your IDE)